### PR TITLE
Add cmake find script for eigen.

### PR DESCRIPTION
**Pull request by schneith**
@trfabian

### CODE REVIEW

**Comment by schneith** [comment]
Would it make sense to unify the two TSIF repos at ASL/RSL?

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,13 +4,14 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -march=native")
 set(TSIF_VERBOSE 0 CACHE STRING "Verbose flag")
 add_definitions(-DTSIF_VERBOSE=${TSIF_VERBOSE})
 
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 find_package(catkin REQUIRED)
-find_package(Eigen3 REQUIRED)
+find_package(Eigen REQUIRED)
```
Comment (trfabian) [inline CMakeLists.txt]: why is this necessary? does the usual find_package not do the job for ubuntu 14.04?

Comment (schneith) [reply CMakeLists.txt]: My 14.04 installation doesn't seem to provide a find script.

**Comment by trfabian** [comment]
@remod i requested a review from you because i'm not sure if this would agree with ANYbotics/RSL policy.

it seems like providing the find script in the package works in more cases compared to relying on the one from the eigen on the system but we usually don't do this...

**Comment by remod** [comment]
I am not in favor of adding support for Ubuntu 14.04 which by itself is not supported anymore... Also, the .cmake file should be forwarded to downstream dependencies using the catkin CFG_EXTRAS and installed with the debian package.

**Comment by trfabian** [comment]
@schneith, since the changes you need are minimal and should be easily maintainable on your end i suggest we keep your fixes for 14.04 on a branch
